build/composer-php-72.json

```
@@ -0,0 +1,29 @@
+{
+    "name": "driftingly/rector-laravel",
+    "type": "rector-extension",
+    "license": "MIT",
+    "description": "Rector upgrades rules for Laravel Framework",
+    "require": {
+        "php": "^7.2 || 8.0.*"
```

```suggestion
        "php": "^7.2 || ^8.0"
```

This allows more project to use it :)